cli/tests/test_git_push.rs

```diff
@@ -2247,6 +2281,8 @@ fn test_git_push_tracked_vs_all(subprocess: bool) {
     ------- stderr -------
     Warning: Non-tracking remote bookmark bookmark1@origin exists
     Hint: Run `jj bookmark track bookmark1@origin` to import the remote bookmark.
+    Warning: Skipping deleted bookmark: bookmark2
+    Hint: Push deleted bookmarks with --deleted.
```

Comment:
This warning probably should be there. It's a niche situation. The bookmark was deleted, but it wasn't tracked, so it also get pushed even with `--deleted`. So the suggestion to use `--deleted` is wrong here, it won't actually do anything.

Reply:
I think the warning condition can be tested in `classify_bookmark_update()`. It's quite similar to `allow_new`.